cmd/rp-rest/static/flightcheckin.html

```diff
@@ -0,0 +1,246 @@
+                    <div class="mb-4">
+                        <label class="block text-grey-darker text-sm font-bold mb-2" for="username">
+                            Booking Number:
+                        </label>
+                        <input class="shadow appearance-none border rounded w-full py-2 px-3 text-grey-darker" id="username" type="text" placeholder="Booking Number">
+                    </div>
+                    <div class="mb-6">
+                        <label class="block text-grey-darker text-sm font-bold mb-2" for="password">
+                           Last Name:
+                        </label>
+                        <input class="shadow appearance-none border rounded w-full py-2 px-3 text-grey-darker" id="username" type="text" placeholder="Last Name">
+                    </div>
+                    <p class="w-48 bg-gray-300 text-center text-gray-800 text-xl font-bold py-4 px-8 border border-blue-900 rounded shadow"
+                       type="submit"> Agree </p>
+
+                </div>
+
+
+        </div>
+    </div>
+
+    <h4 class="w-full p-4 text-2xl font-bold text-center text-red-500" id="error-board"></h4>
+
+    <span id="error"></span>
+    <div class="py-12">
+    </div>
+</div>
+<footer class="text-center gradient">
+    <section class="container mx-auto text-center">
+        <div class="text-sm text-white font py-6">
+            <img class="object-contain object-top-center h-32 w-full" src="img/flight_logo.png">
+            Copyright &copy; <a href="https://securekey.com/"  class="text-white hover:text-green-700" rel="nofollow">SecureKey Technologies</a> and the TrustBloc Contributors.
+        </div>
+    </section>
+</footer>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>
+<script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+<script>
+    function ready(fn) {
+        if (document.readyState !== 'loading'){
+            fn();
+        } else {
+            document.addEventListener('DOMContentLoaded', fn);
+        }
+    }
+    ready(() => {
+        document.getElementById("bookingNumber").innerHTML = randomString();
+        document.getElementById('authBtn').addEventListener('click', performLinkWallet);
+
+    });
+
+    installHandler()
+        .catch(e => console.error('Error in installHandler:', e));
+
+    async function installHandler() {
+        console.log('Loading polyfill...');
+        try {
+            await credentialHandlerPolyfill.loadOnce();
+        } catch (e) {
+            console.error('Error in loadOnce:', e);
+        }
+    }
+
+    async function performLinkWallet() {
+        document.getElementById("error-board").innerText = "";
+
+        const challenge = uuidv4();
+        const domain = window.location.hostname;
+
+        const credentialQuery = {
+            web: {
+                VerifiablePresentation: {
```

Comment:
why we need DID auth for rp?

Reply:
sorry wait .. we dont need this .. wrong file is pushed.  Repushing